app/controllers/admin/customers_controller.rb

```diff
@@ -87,5 +75,9 @@ def user_can_create_customer?
       spree_current_user.admin? ||
         spree_current_user.enterprises.include?(@customer.enterprise)
     end
+
+    def ams_prefix_whitelist
```

Comment:
where's this whitelist used?

Reply:
Used by a method called `render_as_json` to on `Admin::BaseController` which basically sets up a convention whereby a prefix can be submitted in a request and then it is used to find a matching serializer to render the data based on the prefix and the model of the controller. I thought it prudent to maintain a whitelist of serializer prefixes to prevent users from being able to request that data be rendered using some arbitrary serializer.

Reply:
Good call to implement a whitelist.

I could imagine that the ams-prefix is a symptom of a bad API design, but I haven't looked into it much.